dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -281,6 +287,63 @@ private static boolean problemFilter(Map.Entry<SymbolProblem, ClassFile> entry)
     }
   }
 
+  /**
+   * Returns an {@code Optional} describing the linkage error for the interface if any methods of
+   * the interface are not implemented in the referencing {@code classFile}. Such unimplemented
+   * methods manifest as {@link AbstractMethodError} in runtime.
+   */
+  private Optional<SymbolProblem> findInterfaceProblem(
+      ClassFile classFile, InterfaceSymbol interfaceSymbol) {
+    String interfaceName = interfaceSymbol.getClassName();
+    if (classDumper.isSystemClass(interfaceName)) {
+      return Optional.empty();
+    }
+
+    JavaClass implementingClass;
+    JavaClass interfaceDefinition;
```

Comment:
expand the try block so you can keep these two variables inside it.

Reply:
Done.